### PR TITLE
Restore XML hierarchy to WCF configuration section (part 3)

### PR DESCRIPTION
**Pull request by nemrism**
## Summary

Applied XML hierarchy format to WCF configuration section files for:
- `<system.serviceModel.activation><client>...</client></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><comContracts>...</comContracts></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><diagnostics>...</diagnostics></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><protocolMapping>...</protocolMapping></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><routing>...</routing></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><serviceHostingEnvironment>...</serviceHostingEnvironment></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><services>...</services></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><standardEndpoints>...</standardEndpoints></system.serviceModel.activation>` sub-section
- `<system.serviceModel.activation><tracking>...</tracking></system.serviceModel.activation>` sub-section

Contributes to #2038 & #2149

## Remarks

Content of [workflow-for-wcf.md](https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/wcf/workflow-of-wcf.md) file seems a copy-paste of an unaccurate content.

Shouldn't it be replaced with the one form WF section [workflow.md](https://github.com/dotnet/docs/blob/master/docs/framework/configure-apps/file-schema/windows-workflow-foundation/workflow.md) page?

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/participants-of-wcf.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ ms.assetid: d99dbddc-0057-4e18-8e42-f91411d39970
...
   
- For more information in workflow tracking and tracking participants, see [Workflow Tracking and Tracing](../../../windows-workflow-foundation/workflow-tracking-and-tracing.md) and [Tracking Participants](../../../windows-workflow-foundation/tracking-participants.md).  
+For more information in workflow tracking and tracking participants, see [Workflow Tracking and Tracing](../../../windows-workflow-foundation/workflow-tracking-and-tracing.md) and [Tracking Participants](../../../windows-workflow-foundation/tracking-participants.md).  
   
- \<system.serviceModel>  
-\<tracking>  
-\<participants>  
+[**\<configuration>**](../configuration-element.md)\
+&nbsp;&nbsp;[**\<system.serviceModel>**](system-servicemodel.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;[**\<tracking>**](tracking-of-wcf.md)\
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/participants-of-wcf.md]:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;[**\<tracking>**](tracking-of-wcf.md)  
```

File: docs/framework/configure-apps/file-schema/wcf/policyimporter.md [hunk 2]
```diff
@@ -6,11 +6,12 @@ ms.assetid: b0d03456-546f-44bb-ab12-1b2ce7f98fca
...
- \<system.ServiceModel>  
-\<client>  
-\<metadata>  
-\<policyImporters>  
-\<policyImporter>  
+[**\<configuration>**](../configuration-element.md)\
+&nbsp;&nbsp;[**\<system.serviceModel>**](system-servicemodel.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;[**\<client>**](client.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<metadata>**](metadata.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**\<policyImporters>**\
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/policyimporter.md]:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<policyImporters>**](policyimporters.md)  
```

File: docs/framework/configure-apps/file-schema/wcf/wsdlimporter.md [hunk 3]
```diff
@@ -6,11 +6,12 @@ ms.assetid: 986b2165-8430-4dba-b1b8-00396841bb96
...
-\<system.ServiceModel>  
-\<client>  
-\<metadata>  
-\<wsdlImporters>  
-\<wsdlImporter>  
+[**\<configuration>**](../configuration-element.md)\
+&nbsp;&nbsp;[**\<system.serviceModel>**](system-servicemodel.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;[**\<client>**](client.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<metadata>**](metadata.md)\
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**\<wsdlImporters>**\
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/wsdlimporter.md]:
```suggestion
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<wsdlImporters>**](wsdlimporters.md)  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for another PR that improves the formatting of the configuration element hierarchy, @nemrism. I've left a number of suggestions.

**Comment by nemrism** [comment]
Thanks again @rpetrusha!
I've just applied them and added the commits to this PR.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @nemrism. I've resolved a merge conflict, so I'll merge once our internal build completes successfully.

**Pull request merged**